test/t8n/t8n.cpp

```diff
@@ -39,7 +40,7 @@ int main(int argc, const char* argv[])
 
             if (arg == "-v")
             {
-                std::cout << "evmone-t8n " PROJECT_VERSION "\n";
+                std::cout << "evmone-t8n " EVMONE_VERSION "\n";
```

Comment:
What is the difference in output?

Reply:
`PROJECT_VERSION` is not defined any longer.
No we print use `evmone-t8n 0.10.0-dev+commit.f8f94f51` previously it was `evmone-t8n 0.10.0-dev`